src/test/java/lambda/part2/exercise/Exercise1.java

```diff
@@ -0,0 +1,62 @@
+package lambda.part2.exercise;
+
+import java.util.function.BiPredicate;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.function.BiFunction;
+import java.util.function.Function;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+@SuppressWarnings({"unused", "ConstantConditions"})
+public class Exercise1 {
+
+    @Test
+    public void ageExtractorFromPersonUsingMethodReference() {
+        Person person = new Person("Иван", "Мельников", 33);
+
+        Function<Person, Integer> ageExtractor = x -> x.getAge();
+
+        assertEquals(33, ageExtractor.apply(person).intValue());
+    }
+
+    @Test
+    public void sameAgesCheckerUsingBiPredicate() {
+        Person person1 = new Person("Иван", "Мельников", 33);
+        Person person2 = new Person("Дмитрий", "Гущин", 33);
+        Person person3 = new Person("Илья", "Жирков", 22);
+
+        BiPredicate<Person,Person> sameAgesChecker =
+            (x,y) -> x.getAge() == y.getAge();
+
+        assertTrue(sameAgesChecker.test(person1, person2));
+        assertFalse(sameAgesChecker.test(person1, person3));
+        assertFalse(sameAgesChecker.test(person2, person3));
+    }
+
+    private static String getFullName(Person person) {
+        return person.getFirstName() + person.getLastName();
```

Comment:
You forgot 1 space between name and surname

Reply:
Added space between first and last names.